components/Badge/Badge.jsx

```diff
@@ -42,13 +42,15 @@ const BadgeWrapper = styled.div`
 
 const StyledBadge = styled.span`
   border-radius: 8px;
-  display: inline-block;
+  box-sizing: border-box;
+  display: flex;
   font-weight: bold;
-  height: 20px;
-  line-height: 20px;
-  min-width: 20px;
+  height: 24px;
+  line-height: 24px;
+  min-width: 32px;
   text-align: center;
-  box-sizing: border-box;
+  align-items: center;
```

Comment:
i think that we dont need the `text-align: center;` and `align-items: center;`.
i guess this `justify-content: center;` resolves the problem alone

Reply:
done!